R/as_gt.R

```diff
@@ -272,12 +274,12 @@ as_gt.gs_design <- function(
     gt::tab_header(title = parts$title, subtitle = parts$subtitle)
 
   # Add footnotes ----
-  footnote <- parts$footnote
-  for (i in seq_along(footnote$content)) {
-    att <- footnote$attr[i]
+  footnotes <- parts$footnotes
```

Comment:
Hi @yihui, in gsDesign2, we adhere to the singular form in coding, which aligns with our conventions established a few years ago. If you concur, could you please revert "footnotes" to "footnote" for consistency with the other instances? Thank you!